zokrates-worker/src/index.mjs

```diff
@@ -8,10 +10,80 @@ import logger from './utils/logger.mjs';
 
 const {
   MPC: { MPC_PARAMS_URL },
+  DEPLOYMENT_FILES_URL: { DEFAULT_CIRCUIT_FILES_URL },
 } = config;
 
+const { ETH_NETWORK, CIRCUIT_FILES_URL } = process.env;
+
+const finished = promisify(stream.finished);
+
+const downloadFile = async (fileUrl, outputLocationPath) => {
```

Comment:
This method is duplicated, it is a serious candidate to go to a library like `httpUtils`. What do you think?